transport/TransportFactory.cpp

```diff
@@ -460,7 +460,10 @@ class TransportFactoryImpl final : public TransportFactory,
         return metrics;
     }
 
-    bool isGood() const override { return _good; }
+    bool isGood() const override
```

Comment:
what settings do you have for clang-format ?

Reply:
Wrong ones, I guess. I could not run clang-format and used C/C++ extension instead. I will fix clang-format and reformat the code.

Reply:
Fixed.